src/traces/image/plot.js

```diff
@@ -0,0 +1,146 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+var d3 = require('d3');
+var Lib = require('../../lib');
+var xmlnsNamespaces = require('../../constants/xmlns_namespaces');
+var constants = require('./constants');
+
+module.exports = {};
+
+// Generate a function to scale color components according to zmin/zmax and the colormodel
+module.exports.scaler = function(trace) {
```

Comment:
Hmm, do we need to export this function?

Reply:
Not in the current state of the code :sweat_smile:

I do not export it anymore in 73c0740f53fc82a70196d1993ba2824db2be85a3